src/Routes/Index.js

```diff
@@ -43,6 +44,96 @@ class IndexRoute extends BaseRoute {
             });
         });
 
+        this.router.get('/about/manage', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
+            this.db.select().from('about').orderBy('position', 'asc').then((data) => {
+                const sections = data.map((section) => {
+                    section.title = this.renderer.variables(section.title);
+                    section.content = this.renderer.render(section.content);
+                    return section;
+                });
+                res.render('about/manage', { title: 'About', sections });
+            }).catch((e) => {
+                handleError(this.db, req, res, e.stack);
+            });
+        });
+
+        this.router.get('/about/manage/add', this.requiresAuth.bind(this), this.isAdmin.bind(this), (req, res) => {
+            res.render('about/edit', { title: 'Add Section', data: {} });
+        });
+
+        this.router.post('/about/manage/add', this.requiresAuth.bind(this), this.isAdmin.bind(this), async (req, res) => {
+            try {
+                let changes = {};
+                const validate = FormValidator.validateAboutSection(req.body);
+                if (validate && validate.length > 0) return res.render('about/edit', { title: 'Add Section', data: req.body, errors: validate });
+                const columns = Object.keys(await this.db('about').columnInfo());
+                for (const column of columns) {
+                    if (req.body[column]) {
+                        changes[column] = req.body[column];
+                    } else {
+                        changes[column] = null;
+                    }
+                }
+                await this.db('about').insert(changes);
+                res.render('error', { title: 'Success', status: 200, message: 'Section has been added.' });
```

Comment:
Redirect back to the manage page here?

Seems like an annoying flow to add a section and then end up on the error page, with the only CTA being to view lists...